tests/e2e/test_vdbench_workload.py

```diff
@@ -0,0 +1,279 @@
+"""
+Test to exercise VDBench Workload
+"""
+import logging
+import pytest
+import time
+import os
+
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.utility.utils import TimeoutSampler, ocsci_log_path
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.utility import templating
+from ocs_ci.ocs.utils import get_pod_name_by_pattern
+from ocs_ci.ocs.ripsaw import RipSaw
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import E2ETest, workloads
+from tests import helpers
+
+log = logging.getLogger(__name__)
+
+
+# TODO: move this function to pod.py helper functions
+def download(pod_name, localpath, remotepath):
+    """
+    Download a file from pod
+
+    Args:
+        pod_name (str): Name of the pod
+        localpath (str): Local file to download
+        remotepath (str): Target path on the pod
+
+    """
+    cmd = f'cp {pod_name}:{remotepath} {os.path.expanduser(localpath)}'
+    ocp_obj = OCP()
+    ocp_obj.exec_oc_cmd(command=cmd)
+
+
+# TODO: move this function to helper.py helper functions
+def get_labeled_nodes(label):
+    """
+    Fetches all worker nodes.
+
+    Args:
+        label (str): node label to look for
+    Returns:
+        list: List of names of labeled nodes
+    """
+    ocp_node_obj = OCP(kind=constants.NODE)
+    nodes = ocp_node_obj.get(selector=label).get('items')
+    labeled_nodes_list = [node.get('metadata').get('name') for node in nodes]
+    return labeled_nodes_list
+
+
+def label_node(node, label):
+    """
+    Add / Delete label to / from node.
+
+    Args:
+        node (str): node name
+        label (str): the label that need to be add/delete to/from the node
+    """
+    ocp_obj = OCP()
+    ocp_obj.exec_oc_cmd(
+        command=f'label node {node} {label}', out_yaml_format=False
+    )
+
+
+@pytest.fixture(scope='function')
+def label_nodes(request):
```

Comment:
can you pls update the docstring with arg and return value type

Reply:
This fixture have no arguments. it just set app-node label to non-ocs workers.